src/coreclr/jit/fgdiagnostic.cpp

```diff
@@ -645,44 +580,49 @@ FILE* Compiler::fgOpenFlowGraphFile(bool* wbDontClose, Phases phase, LPCWSTR typ
 //    The xml dumps are the historical mechanism for dumping the flowgraph.
 //    The dot format can be viewed by:
 //    - Graphviz (http://www.graphviz.org/)
-//      - The command "C:\Program Files (x86)\Graphviz2.38\bin\dot.exe" -Tsvg -oFoo.svg -Kdot Foo.dot
-//        will produce a Foo.svg file that can be opened with any svg-capable browser (e.g. IE).
+//      - The command:
+//           "C:\Program Files (x86)\Graphviz2.38\bin\dot.exe" -Tsvg -oFoo.svg -Kdot Foo.dot
+//        will produce a Foo.svg file that can be opened with any svg-capable browser.
+//    - https://sketchviz.com/
 //    - http://rise4fun.com/Agl/
 //      - Cut and paste the graph from your .dot file, replacing the digraph on the page, and then click the play
 //        button.
 //      - It will show a rotating '/' and then render the graph in the browser.
-//    MSAGL has also been open-sourced to https://github.com/Microsoft/automatic-graph-layout.git.
+//    MSAGL has also been open-sourced to https://github.com/Microsoft/automatic-graph-layout.
```

Comment:
Last I looked MSAGL no longer included a dot parser.

Reply:
Interesting. Well, it's still useful to have a link.